spec/classes/init_spec.rb

```diff
@@ -971,11 +971,15 @@
   end
 
   describe 'sshd_config_permittunnel param' do
-    ['yes','point-to-point','ethernet','no'].each do |value|
+    ['yes','point-to-point','ethernet','no','unset'].each do |value|
       context "set to #{value}" do
         let (:params) { { :sshd_config_permittunnel => value } }
 
-        it { should contain_file('sshd_config').with_content(/^PermitTunnel #{value}$/) }
+        if value == 'unset'
+          it { should contain_file('sshd_config').without_content(/^PermitTunnel/) }
```

Comment:
negative matches should have `/^\s*` as the start of the regex to protect us from whitespace shifting in the template.